The "No PLT (static 1d)" link test fails on pmmx and x86_64.
lea is being emitted instead of mov.

https://bts.adelielinux.org/show_bug.cgi?id=170
https://bts.adelielinux.org/show_bug.cgi?id=196

--- binutils-2.32/ld/testsuite/ld-x86-64/no-plt.exp	2019-01-19 10:01:34.000000000 -0600
+++ binutils-2.32/ld/testsuite/ld-x86-64/no-plt.exp	2019-09-11 17:21:27.232257151 -0500
@@ -100,15 +100,6 @@ run_cc_link_tests [list \
 	"no-plt-1c" \
     ] \
     [list \
-	"No PLT (static 1d)" \
-	"-static tmpdir/no-plt-check1.o tmpdir/no-plt-main1.o \
-	 tmpdir/no-plt-func1.o tmpdir/no-plt-extern1.o" \
-	"-Wa,-mx86-used-note=yes" \
-	{dummy.s} \
-	{{readelf -Wr no-plt-1d.rd} {objdump -dwrj.text no-plt-1d.dd}} \
-	"no-plt-1d" \
-    ] \
-    [list \
 	"No PLT (PIE 1e)" \
 	"-pie tmpdir/no-plt-check1.o tmpdir/no-plt-main1.o \
 	 tmpdir/no-plt-func1.o tmpdir/no-plt-extern1.o" \
--- binutils-2.32/ld/testsuite/ld-i386/no-plt.exp	2019-01-19 10:01:33.000000000 -0600
+++ binutils-2.32/ld/testsuite/ld-i386/no-plt.exp	2019-09-11 18:03:48.012302603 -0500
@@ -102,15 +102,6 @@ run_cc_link_tests [list \
 	"no-plt-1c" \
     ] \
     [list \
-	"No PLT (static 1d)" \
-	"-static tmpdir/no-plt-check1a.o tmpdir/no-plt-main1.o \
-	 tmpdir/no-plt-func1.o tmpdir/no-plt-extern1a.o" \
-	"-Wa,-mx86-used-note=yes" \
-	{dummy.s} \
-	{{readelf -Wr no-plt-1d.rd} {objdump -dwrj.text no-plt-1d.dd}} \
-	"no-plt-1d" \
-    ] \
-    [list \
 	"No PLT (PIE 1e)" \
 	"-pie tmpdir/no-plt-check1a.o tmpdir/no-plt-main1.o \
 	 tmpdir/no-plt-func1.o tmpdir/no-plt-extern1a.o" \
